default/python/AI/ColonisationAI.py

```diff
@@ -218,6 +217,7 @@ def assign_colonisation_values(planet_ids, mission_type, species, detail=None, r
     for planet_id in planet_ids:
         pv = []
         for spec_name in try_species:
+            # TODO: this function never returned detail to its caller, can we remove it?
```

Comment:
I know and calculate_planet_colonization_rating still does it and I am using the values.
But while assign_colonisation_values uses the same argument, it never forwarded the details, as far as I could see.
I haven't removed it yet because I may find it useful one day, when I work in the functions calling assign_colonisation_values.